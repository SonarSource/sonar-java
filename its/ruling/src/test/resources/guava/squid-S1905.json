{
'com.google.guava:guava:src/com/google/common/collect/Maps.java':[
654,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableSortedMultiset.java':[
114,
],
'com.google.guava:guava:src/com/google/common/collect/SortedIterables.java':[
56,
],
'com.google.guava:guava:src/com/google/common/net/InetAddresses.java':[
707,
],
}
