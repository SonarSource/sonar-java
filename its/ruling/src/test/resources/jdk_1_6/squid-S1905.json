{
'jdk_1_6:java/applet/Applet.java':[
103,
],
'jdk_1_6:java/awt/Component.java':[
4797,
7475,
],
'jdk_1_6:java/awt/Container.java':[
3053,
4459,
4560,
],
'jdk_1_6:java/awt/Cursor.java':[
288,
292,
],
'jdk_1_6:java/awt/DefaultKeyboardFocusManager.java':[
511,
705,
940,
],
'jdk_1_6:java/awt/Dialog.java':[
1391,
],
'jdk_1_6:java/awt/Font.java':[
2396,
2415,
2435,
2455,
],
'jdk_1_6:java/awt/KeyboardFocusManager.java':[
2227,
2232,
2628,
2631,
],
'jdk_1_6:java/awt/List.java':[
476,
],
'jdk_1_6:java/awt/MenuComponent.java':[
279,
292,
],
'jdk_1_6:java/awt/MultipleGradientPaintContext.java':[
648,
670,
],
'jdk_1_6:java/awt/Toolkit.java':[
1941,
],
'jdk_1_6:java/awt/Window.java':[
458,
490,
],
'jdk_1_6:java/awt/datatransfer/DataFlavor.java':[
266,
],
'jdk_1_6:java/awt/datatransfer/StringSelection.java':[
112,
],
'jdk_1_6:java/awt/dnd/DragGestureRecognizer.java':[
280,
],
'jdk_1_6:java/awt/image/BandedSampleModel.java':[
264,
283,
301,
319,
337,
],
'jdk_1_6:java/awt/image/ComponentColorModel.java':[
1500,
1506,
],
'jdk_1_6:java/awt/image/ComponentSampleModel.java':[
597,
615,
632,
649,
666,
],
'jdk_1_6:java/awt/image/DataBufferInt.java':[
207,
],
'jdk_1_6:java/awt/image/DataBufferUShort.java':[
203,
216,
],
'jdk_1_6:java/awt/image/IndexColorModel.java':[
894,
],
'jdk_1_6:java/awt/image/MultiPixelPackedSampleModel.java':[
183,
461,
477,
493,
],
'jdk_1_6:java/awt/image/PixelGrabber.java':[
312,
313,
],
'jdk_1_6:java/awt/image/SampleModel.java':[
364,
388,
411,
434,
457,
],
'jdk_1_6:java/awt/image/SinglePixelPackedSampleModel.java':[
379,
393,
407,
735,
],
'jdk_1_6:java/awt/image/renderable/ParameterBlock.java':[
152,
],
'jdk_1_6:java/beans/Beans.java':[
289,
],
'jdk_1_6:java/beans/FeatureDescriptor.java':[
300,
],
'jdk_1_6:java/beans/beancontext/BeanContextSupport.java':[
738,
],
'jdk_1_6:java/lang/reflect/Constructor.java':[
192,
],
'jdk_1_6:java/lang/reflect/Method.java':[
198,
],
'jdk_1_6:java/lang/reflect/Proxy.java':[
588,
],
'jdk_1_6:java/net/InetAddress.java':[
817,
819,
],
'jdk_1_6:java/net/PlainSocketImpl.java':[
309,
],
'jdk_1_6:java/net/URLConnection.java':[
1592,
],
'jdk_1_6:java/nio/Bits.java':[
36,
41,
193,
],
'jdk_1_6:java/nio/DirectByteBuffer.java':[
508,
514,
522,
528,
599,
605,
613,
619,
690,
696,
704,
710,
781,
787,
795,
801,
872,
878,
886,
892,
963,
969,
977,
983,
],
'jdk_1_6:java/nio/DirectByteBufferR.java':[
508,
514,
522,
528,
599,
605,
613,
619,
690,
696,
704,
710,
781,
787,
795,
801,
872,
878,
886,
892,
963,
969,
977,
983,
],
'jdk_1_6:java/nio/HeapByteBuffer.java':[
265,
271,
316,
322,
367,
373,
418,
424,
469,
475,
520,
526,
],
'jdk_1_6:java/nio/HeapByteBufferR.java':[
265,
271,
316,
322,
367,
373,
418,
424,
469,
475,
520,
526,
],
'jdk_1_6:java/nio/IntBuffer.java':[
1064,
],
'jdk_1_6:java/rmi/dgc/VMID.java':[
104,
],
'jdk_1_6:java/security/MessageDigest.java':[
520,
521,
522,
],
'jdk_1_6:java/security/Permissions.java':[
226,
434,
],
'jdk_1_6:java/security/Provider.java':[
588,
609,
630,
],
'jdk_1_6:java/security/SecureClassLoader.java':[
188,
],
'jdk_1_6:java/security/Security.java':[
586,
],
'jdk_1_6:java/security/Signature.java':[
919,
920,
],
'jdk_1_6:java/security/cert/X509CRLSelector.java':[
616,
],
'jdk_1_6:java/security/cert/X509CertSelector.java':[
2173,
2189,
2289,
2325,
2393,
2462,
2464,
],
'jdk_1_6:java/text/DecimalFormat.java':[
1343,
1343,
],
'jdk_1_6:java/util/ArrayList.java':[
305,
],
'jdk_1_6:java/util/Arrays.java':[
1247,
2734,
2760,
3027,
],
'jdk_1_6:java/util/EnumMap.java':[
243,
],
'jdk_1_6:java/util/GregorianCalendar.java':[
575,
649,
2096,
2806,
2894,
],
'jdk_1_6:java/util/HashSet.java':[
282,
],
'jdk_1_6:java/util/PriorityQueue.java':[
159,
162,
185,
203,
400,
],
'jdk_1_6:java/util/Properties.java':[
565,
],
'jdk_1_6:java/util/Random.java':[
492,
],
'jdk_1_6:java/util/RegularEnumSet.java':[
144,
],
'jdk_1_6:java/util/TreeSet.java':[
288,
],
'jdk_1_6:java/util/Vector.java':[
671,
],
'jdk_1_6:java/util/WeakHashMap.java':[
213,
],
'jdk_1_6:java/util/concurrent/ConcurrentSkipListMap.java':[
950,
],
'jdk_1_6:java/util/concurrent/CopyOnWriteArrayList.java':[
326,
],
'jdk_1_6:java/util/concurrent/atomic/AtomicLong.java':[
240,
],
'jdk_1_6:java/util/jar/Manifest.java':[
109,
],
'jdk_1_6:java/util/logging/Logger.java':[
1392,
1470,
],
'jdk_1_6:java/util/prefs/FileSystemPreferences.java':[
152,
],
'jdk_1_6:java/util/zip/GZIPInputStream.java':[
237,
],
'jdk_1_6:java/util/zip/ZipInputStream.java':[
302,
309,
310,
],
}
