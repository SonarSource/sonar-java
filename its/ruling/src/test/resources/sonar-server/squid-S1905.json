{
'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/activity/index/ActivityDoc.java':[
70,
],
'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/issue/index/IssueIndex.java':[
660,
],
'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/qualityprofile/QProfileActivity.java':[
84,
],
'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/qualityprofile/index/ActiveRuleDoc.java':[
61,
],
'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/rule/index/RuleDoc.java':[
111,
134,
139,
195,
201,
207,
213,
218,
223,
244,
249,
270,
275,
283,
295,
],
'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/source/index/SourceLineDoc.java':[
62,
185,
],
'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/user/index/UserDoc.java':[
63,
],
}
