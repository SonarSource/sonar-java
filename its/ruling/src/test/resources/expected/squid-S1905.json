{
'project:com/sun/corba/se/PortableActivationIDL/EndPointInfo.java':[
14,
],
'project:com/sun/corba/se/PortableActivationIDL/ORBPortInfo.java':[
14,
],
'project:com/sun/corba/se/PortableActivationIDL/TCPPortHelper.java':[
52,
],
'project:com/sun/corba/se/PortableActivationIDL/_LocatorImplBase.java':[
101,
121,
],
'project:com/sun/corba/se/PortableActivationIDL/_ServerManagerImplBase.java':[
341,
361,
],
'project:com/sun/corba/se/impl/activation/ServerTool.java':[
92,
579,
],
'project:com/sun/corba/se/impl/corba/AnyImpl.java':[
525,
],
'project:com/sun/corba/se/impl/corba/TypeCodeImpl.java':[
645,
],
'project:com/sun/corba/se/impl/encoding/CDRInputStream_1_0.java':[
1893,
1894,
],
'project:com/sun/corba/se/impl/encoding/CDROutputStream_1_0.java':[
750,
],
'project:com/sun/corba/se/impl/encoding/IDLJavaSerializationInputStream.java':[
597,
598,
],
'project:com/sun/corba/se/impl/interceptors/ClientRequestInfoImpl.java':[
645,
],
'project:com/sun/corba/se/impl/interceptors/InterceptorList.java':[
127,
],
'project:com/sun/corba/se/impl/interceptors/PICurrent.java':[
99,
],
'project:com/sun/corba/se/impl/io/IIOPInputStream.java':[
327,
1998,
2004,
2009,
],
'project:com/sun/corba/se/impl/io/ObjectStreamClass.java':[
673,
775,
777,
],
'project:com/sun/corba/se/impl/io/ValueHandlerImpl.java':[
264,
299,
480,
485,
490,
495,
500,
505,
510,
515,
525,
532,
618,
622,
626,
630,
634,
638,
642,
646,
683,
748,
756,
762,
783,
],
'project:com/sun/corba/se/impl/ior/StubIORImpl.java':[
119,
],
'project:com/sun/corba/se/impl/naming/cosnaming/NamingContextImpl.java':[
165,
210,
255,
306,
349,
387,
414,
438,
519,
869,
898,
947,
986,
],
'project:com/sun/corba/se/impl/naming/pcosnaming/NamingContextImpl.java':[
214,
249,
287,
324,
359,
387,
408,
429,
495,
],
'project:com/sun/corba/se/impl/oa/poa/AOMEntry.java':[
212,
219,
],
'project:com/sun/corba/se/impl/oa/poa/POAPolicyMediatorBase.java':[
87,
],
'project:com/sun/corba/se/impl/oa/toa/TOAImpl.java':[
87,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClass_1_3_1.java':[
648,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryId_1_3.java':[
91,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryId_1_3_1.java':[
95,
],
'project:com/sun/corba/se/impl/orbutil/closure/Future.java':[
38,
],
'project:com/sun/corba/se/impl/presentation/rmi/DynamicStubImpl.java':[
96,
],
'project:com/sun/corba/se/impl/presentation/rmi/IDLNameTranslatorImpl.java':[
113,
],
'project:com/sun/corba/se/impl/presentation/rmi/StubConnectImpl.java':[
77,
],
'project:com/sun/corba/se/impl/protocol/CorbaClientDelegateImpl.java':[
161,
174,
177,
231,
250,
],
'project:com/sun/corba/se/impl/protocol/CorbaMessageMediatorImpl.java':[
188,
192,
198,
674,
873,
893,
915,
963,
985,
1033,
],
'project:com/sun/corba/se/impl/protocol/CorbaServerRequestDispatcherImpl.java':[
654,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/CancelRequestMessage_1_0.java':[
41,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/CancelRequestMessage_1_1.java':[
41,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/CancelRequestMessage_1_2.java':[
41,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/IORAddressingInfo.java':[
38,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/LocateReplyMessage_1_0.java':[
56,
57,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/LocateReplyMessage_1_1.java':[
55,
56,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/LocateReplyMessage_1_2.java':[
62,
65,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/LocateRequestMessage_1_0.java':[
45,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/LocateRequestMessage_1_1.java':[
46,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/MessageBase.java':[
84,
409,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/Message_1_0.java':[
49,
53,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/Message_1_1.java':[
53,
57,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/Message_1_2.java':[
33,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/ReplyMessage_1_0.java':[
61,
62,
65,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/ReplyMessage_1_1.java':[
61,
62,
65,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/ReplyMessage_1_2.java':[
64,
68,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/RequestMessage_1_0.java':[
47,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/RequestMessage_1_1.java':[
51,
],
'project:com/sun/corba/se/impl/resolver/BootstrapResolverImpl.java':[
99,
],
'project:com/sun/corba/se/impl/util/RepositoryId.java':[
89,
],
'project:com/sun/corba/se/spi/activation/EndPointInfo.java':[
14,
],
'project:com/sun/corba/se/spi/activation/ORBPortInfo.java':[
14,
],
'project:com/sun/corba/se/spi/activation/ServerAlreadyActive.java':[
13,
],
'project:com/sun/corba/se/spi/activation/ServerAlreadyInstalled.java':[
13,
],
'project:com/sun/corba/se/spi/activation/ServerAlreadyRegistered.java':[
13,
],
'project:com/sun/corba/se/spi/activation/ServerAlreadyUninstalled.java':[
13,
],
'project:com/sun/corba/se/spi/activation/ServerHeldDown.java':[
13,
],
'project:com/sun/corba/se/spi/activation/ServerIdHelper.java':[
46,
],
'project:com/sun/corba/se/spi/activation/ServerNotActive.java':[
13,
],
'project:com/sun/corba/se/spi/activation/ServerNotRegistered.java':[
13,
],
'project:com/sun/corba/se/spi/activation/TCPPortHelper.java':[
46,
],
'project:com/sun/corba/se/spi/activation/_LocatorImplBase.java':[
94,
112,
],
'project:com/sun/corba/se/spi/activation/_RepositoryImplBase.java':[
51,
178,
],
'project:com/sun/corba/se/spi/activation/_ServerManagerImplBase.java':[
250,
268,
],
'project:com/sun/corba/se/spi/presentation/rmi/StubAdapter.java':[
158,
],
'project:com/sun/imageio/plugins/bmp/BMPImageReader.java':[
770,
773,
776,
],
'project:com/sun/imageio/plugins/bmp/BMPImageWriter.java':[
585,
],
'project:com/sun/imageio/plugins/common/ImageUtil.java':[
596,
605,
838,
],
'project:com/sun/imageio/plugins/common/LZWStringTable.java':[
146,
],
'project:com/sun/imageio/plugins/jpeg/JPEGMetadata.java':[
726,
2018,
2025,
],
'project:com/sun/imageio/plugins/png/PNGMetadata.java':[
766,
],
'project:com/sun/imageio/plugins/wbmp/WBMPImageWriter.java':[
287,
],
'project:com/sun/java/swing/plaf/windows/WindowsMenuBarUI.java':[
145,
146,
],
'project:com/sun/jmx/remote/security/MBeanServerFileAccessController.java':[
286,
],
'project:com/sun/jmx/snmp/IPAcl/AclEntryImpl.java':[
99,
],
'project:com/sun/jmx/snmp/daemon/SnmpAdaptorServer.java':[
485,
],
'project:com/sun/mirror/util/SourceOrderDeclScanner.java':[
123,
],
'project:com/sun/org/apache/bcel/internal/classfile/Attribute.java':[
165,
],
'project:com/sun/org/apache/bcel/internal/generic/InstructionFactory.java':[
303,
304,
320,
321,
370,
371,
378,
379,
386,
387,
394,
395,
],
'project:com/sun/org/apache/bcel/internal/util/CodeHTML.java':[
197,
436,
],
'project:com/sun/org/apache/xalan/internal/lib/ExsltDatetime.java':[
621,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/EqualityExpr.java':[
200,
201,
210,
211,
215,
216,
272,
273,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/FunctionCall.java':[
527,
847,
848,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Parser.java':[
419,
583,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/TestSeq.java':[
200,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/XPathParser.java':[
1382,
1952,
1970,
1988,
2013,
2765,
2818,
2838,
2879,
2896,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/BooleanType.java':[
207,
208,
212,
213,
217,
218,
222,
223,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/IntType.java':[
284,
285,
289,
290,
294,
295,
299,
300,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/KeyIndex.java':[
376,
412,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SAXImpl.java':[
452,
1554,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SAX2DOM.java':[
168,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TransformerImpl.java':[
348,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDTDScannerImpl.java':[
334,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityManager.java':[
941,
2805,
2813,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLScanner.java':[
354,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLStreamReaderImpl.java':[
267,
609,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/util/Base64.java':[
82,
83,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/BaseDVFactory.java':[
72,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/ExtendedSchemaDVFactoryImpl.java':[
84,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/FullDVFactory.java':[
69,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/SchemaDVFactoryImpl.java':[
74,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/XSSimpleTypeDecl.java':[
436,
547,
2599,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaValidator.java':[
2749,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSComplexTypeDecl.java':[
247,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSGrammarBucket.java':[
55,
98,
171,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/CMBuilder.java':[
433,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDHandler.java':[
637,
949,
1333,
1463,
3681,
4006,
4042,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDUniqueOrKeyTraverser.java':[
92,
],
'project:com/sun/org/apache/xerces/internal/jaxp/SAXParserImpl.java':[
230,
350,
354,
358,
],
'project:com/sun/org/apache/xerces/internal/util/ParserConfigurationSettings.java':[
209,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeHandler.java':[
1646,
1677,
],
'project:com/sun/org/apache/xml/internal/dtm/DTMException.java':[
361,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBase.java':[
645,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMManagerDefault.java':[
575,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMNodeList.java':[
78,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMNodeProxy.java':[
327,
768,
896,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/IncrementalSAXSource_Xerces.java':[
227,
238,
256,
],
'project:com/sun/org/apache/xml/internal/resolver/Catalog.java':[
1623,
1739,
1765,
1790,
1897,
1923,
1948,
],
'project:com/sun/org/apache/xml/internal/serialize/BaseMarkupSerializer.java':[
536,
1579,
],
'project:com/sun/org/apache/xml/internal/serialize/XML11Serializer.java':[
223,
351,
441,
463,
495,
517,
],
'project:com/sun/org/apache/xml/internal/serialize/XMLSerializer.java':[
1296,
1318,
1350,
1372,
],
'project:com/sun/org/apache/xml/internal/utils/DOMBuilder.java':[
116,
116,
],
'project:com/sun/org/apache/xpath/internal/NodeSet.java':[
354,
586,
652,
],
'project:com/sun/org/apache/xpath/internal/axes/LocPathIterator.java':[
298,
],
'project:com/sun/org/apache/xpath/internal/compiler/Lexer.java':[
591,
613,
],
'project:com/sun/org/apache/xpath/internal/compiler/OpMap.java':[
450,
],
'project:com/sun/security/auth/module/LdapLoginModule.java':[
746,
876,
926,
],
'project:java/applet/Applet.java':[
120,
],
'project:java/awt/Component.java':[
5008,
7686,
],
'project:java/awt/Container.java':[
4715,
4817,
],
'project:java/awt/Cursor.java':[
310,
314,
],
'project:java/awt/DefaultKeyboardFocusManager.java':[
558,
746,
967,
],
'project:java/awt/Dialog.java':[
1394,
],
'project:java/awt/Font.java':[
2492,
2511,
2531,
2551,
],
'project:java/awt/KeyboardFocusManager.java':[
2282,
2288,
2624,
2628,
],
'project:java/awt/List.java':[
493,
],
'project:java/awt/MenuComponent.java':[
291,
304,
],
'project:java/awt/Window.java':[
561,
593,
],
'project:java/awt/datatransfer/DataFlavor.java':[
283,
],
'project:java/awt/datatransfer/StringSelection.java':[
130,
],
'project:java/awt/dnd/DragGestureRecognizer.java':[
300,
],
'project:java/awt/image/BandedSampleModel.java':[
282,
301,
319,
337,
355,
],
'project:java/awt/image/ComponentColorModel.java':[
1517,
1523,
],
'project:java/awt/image/ComponentSampleModel.java':[
615,
633,
650,
667,
684,
],
'project:java/awt/image/DataBufferInt.java':[
281,
],
'project:java/awt/image/DataBufferUShort.java':[
275,
288,
],
'project:java/awt/image/IndexColorModel.java':[
911,
],
'project:java/awt/image/MultiPixelPackedSampleModel.java':[
201,
479,
495,
511,
],
'project:java/awt/image/PixelGrabber.java':[
329,
330,
],
'project:java/awt/image/SampleModel.java':[
391,
415,
438,
461,
484,
],
'project:java/awt/image/SinglePixelPackedSampleModel.java':[
383,
397,
411,
749,
],
'project:java/awt/image/renderable/ParameterBlock.java':[
170,
],
'project:java/beans/Beans.java':[
315,
],
'project:java/beans/beancontext/BeanContextSupport.java':[
755,
],
'project:java/lang/invoke/AdapterMethodHandle.java':[
392,
393,
415,
425,
426,
],
'project:java/lang/invoke/MethodHandleProxies.java':[
150,
151,
],
'project:java/lang/invoke/MethodHandleStatics.java':[
65,
],
'project:java/lang/invoke/ToGeneric.java':[
396,
],
'project:java/net/AbstractPlainSocketImpl.java':[
296,
],
'project:java/nio/DirectByteBuffer.java':[
539,
545,
553,
559,
630,
636,
644,
650,
721,
727,
735,
741,
812,
818,
826,
832,
903,
909,
917,
923,
994,
1000,
1008,
1014,
],
'project:java/nio/DirectByteBufferR.java':[
539,
545,
553,
559,
630,
636,
644,
650,
721,
727,
735,
741,
812,
818,
826,
832,
903,
909,
917,
923,
994,
1000,
1008,
1014,
],
'project:java/nio/HeapByteBuffer.java':[
284,
290,
335,
341,
386,
392,
437,
443,
488,
494,
539,
545,
],
'project:java/nio/HeapByteBufferR.java':[
284,
290,
335,
341,
386,
392,
437,
443,
488,
494,
539,
545,
],
'project:java/nio/IntBuffer.java':[
1083,
],
'project:java/rmi/dgc/VMID.java':[
121,
],
'project:java/security/MessageDigest.java':[
550,
551,
552,
],
'project:java/security/Signature.java':[
951,
952,
],
'project:java/text/DecimalFormat.java':[
1362,
1362,
],
'project:java/util/ArrayList.java':[
360,
],
'project:java/util/Arrays.java':[
2219,
2245,
2245,
2512,
2512,
],
'project:java/util/GregorianCalendar.java':[
620,
694,
2348,
3086,
3174,
],
'project:java/util/HashSet.java':[
299,
],
'project:java/util/PriorityQueue.java':[
177,
182,
207,
226,
455,
],
'project:java/util/TreeSet.java':[
305,
],
'project:java/util/Vector.java':[
716,
],
'project:java/util/concurrent/CopyOnWriteArrayList.java':[
355,
],
'project:java/util/concurrent/ForkJoinPool.java':[
1644,
],
'project:java/util/concurrent/PriorityBlockingQueue.java':[
240,
246,
868,
],
'project:java/util/concurrent/ScheduledThreadPoolExecutor.java':[
1232,
],
'project:java/util/zip/GZIPInputStream.java':[
255,
],
'project:javax/accessibility/AccessibleRelationSet.java':[
198,
218,
234,
237,
],
'project:javax/accessibility/AccessibleStateSet.java':[
174,
190,
193,
],
'project:javax/imageio/IIOImage.java':[
261,
],
'project:javax/imageio/ImageReader.java':[
682,
2018,
2036,
2056,
2077,
2095,
2118,
2139,
2157,
2175,
2211,
2252,
2277,
2314,
2356,
2382,
2408,
2453,
2454,
2871,
],
'project:javax/imageio/ImageWriter.java':[
1759,
1780,
1798,
1821,
1842,
1860,
1878,
1907,
1955,
1956,
],
'project:javax/imageio/metadata/IIOMetadataFormatImpl.java':[
1181,
1191,
],
'project:javax/management/modelmbean/RequiredModelMBean.java':[
1089,
],
'project:javax/management/monitor/Monitor.java':[
897,
],
'project:javax/management/openmbean/CompositeType.java':[
229,
],
'project:javax/management/openmbean/OpenMBeanAttributeInfoSupport.java':[
470,
537,
936,
938,
],
'project:javax/management/remote/rmi/RMIConnectionImpl_Stub.java':[
254,
402,
],
'project:javax/management/timer/Timer.java':[
312,
318,
483,
484,
487,
1101,
1107,
],
'project:javax/naming/Reference.java':[
228,
244,
],
'project:javax/print/attribute/HashAttributeSet.java':[
385,
],
'project:javax/rmi/CORBA/Stub.java':[
67,
],
'project:javax/rmi/PortableRemoteObject.java':[
198,
],
'project:javax/sound/midi/MidiSystem.java':[
470,
],
'project:javax/sound/midi/Sequence.java':[
315,
],
'project:javax/sound/sampled/AudioSystem.java':[
1610,
],
'project:javax/sql/rowset/serial/SQLInputImpl.java':[
282,
594,
623,
626,
],
'project:javax/sql/rowset/serial/SerialBlob.java':[
210,
],
'project:javax/sql/rowset/serial/SerialClob.java':[
199,
],
'project:javax/swing/DebugGraphics.java':[
1392,
],
'project:javax/swing/JWindow.java':[
211,
249,
],
'project:javax/swing/plaf/basic/BasicMenuBarUI.java':[
231,
232,
233,
],
'project:javax/xml/transform/TransformerException.java':[
345,
],
'project:org/omg/CORBA/PolicyTypeHelper.java':[
75,
],
'project:org/omg/CORBA/StringValueHelper.java':[
115,
],
'project:org/omg/CORBA/WStringValueHelper.java':[
119,
],
'project:org/omg/IOP/ComponentIdHelper.java':[
51,
],
'project:org/omg/IOP/ProfileIdHelper.java':[
48,
],
'project:org/omg/IOP/ServiceContext.java':[
15,
],
'project:org/omg/IOP/ServiceIdHelper.java':[
48,
],
'project:org/omg/IOP/TaggedComponent.java':[
15,
],
'project:org/omg/IOP/TaggedProfile.java':[
15,
],
'project:org/omg/PortableInterceptor/AdapterManagerIdHelper.java':[
53,
],
'project:org/w3c/dom/bootstrap/DOMImplementationRegistry.java':[
379,
],
}
